src/pages/tasks/TaskAssigneeSelectorModal.js

```diff
@@ -201,7 +201,7 @@ function TaskAssigneeSelectorModal(props) {
         }
     };
 
-    const isOpen = ReportUtils.isOpenTaskReport(props.task.report);
+    const isOpen = report && ReportUtils.isOpenTaskReport(report);
```

Comment:
Isn't `report && ...` part redundant? 
Inside ReportUtils.isOpenTaskReport -> isTaskReport we are already checking it as first condition: 
https://github.com/Expensify/App/blob/1998fa35e7bcc217ca29757efb7a9fd364346988/src/libs/ReportUtils.js#L190